securedrop_client/gui/widgets.py

```diff
@@ -1776,7 +1776,8 @@ def __init__(
         message_failed_signal.connect(self._on_reply_failure)
 
         # Set styles
```

Comment:
nit: this comment seems out of place (i know this wasn't introduced by your code but just noticed it above your new lines of code and was like, this isn't setting styles!)